web-ui/src/main/resources/catalog/components/userfeedback/partials/userfeedbacknew.html

```diff
@@ -0,0 +1,226 @@
+<div>
```

Comment:
Indentation of html files looks not fine, should be used 2 characters. Please make sure to use an IDE that uses `.editorconfig`

Reply:
Fixed